source/_patterns/05-components/form-item/_form-item.scss

```diff
@@ -37,63 +15,3 @@ $form-text-size: gesso-base-font-size() !default;
     margin-bottom: 0;
   }
 }
-
-.form-item__required-marker::before {
-  content: '*';
-}
```

Comment:
We need this CSS to mark required fields, or else update `templates/form/form-element-label.html.twig` to not wrap the required text with `visually-hidden`.

Reply:
The way USWDS handles this, which you can see in the text inputs in PL (I think that's where I demoed it), is the reverse: non-required fields are marked (optional). I'm inclined to follow their lead here, which it sounds like means updating form-element-label.html.twig.

Reply:
I like that approach. 👍